src/main/java/org/aion/wallet/ui/components/SendPane.java

```diff
@@ -0,0 +1,171 @@
+package org.aion.wallet.ui.components;
+
+import javafx.application.Platform;
+import javafx.collections.FXCollections;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.Label;
+import javafx.scene.control.PasswordField;
+import javafx.scene.control.TextField;
+import javafx.scene.input.MouseEvent;
+import org.aion.base.util.TypeConverter;
+import org.aion.log.AionLoggerFactory;
+import org.aion.log.LogEnum;
+import org.aion.wallet.connector.BlockchainConnector;
+import org.aion.wallet.connector.WalletBlockchainConnector;
+import org.aion.wallet.connector.dto.SendRequestDTO;
+import org.aion.zero.types.AionTransaction;
+import org.slf4j.Logger;
+
+import java.math.BigDecimal;
+import java.net.URL;
+import java.util.List;
+import java.util.ResourceBundle;
+import java.util.Timer;
+import java.util.TimerTask;
+
+import static org.aion.wallet.util.BalanceFormatter.WEI_MULTIPLIER;
+
+public class SendPane implements Initializable {
+    private static final Logger log = AionLoggerFactory.getLogger(LogEnum.WLT.name());
+    private final BlockchainConnector blockchainConnector = new WalletBlockchainConnector();
+    private final static Long DEFAULT_BLOCK_MINING_TIME = 10000L;
+    private final static int MAX_TX_STATUS_RETRY_COUNT = 6;
+
+    @FXML
+    private ComboBox fromInput;
+    @FXML
+    private PasswordField passwordInput;
+    @FXML
+    private TextField toInput;
+    @FXML
+    private TextField nrgInput;
+    @FXML
+    private TextField nrgPriceInput;
+    @FXML
+    private TextField valueInput;
+    @FXML
+    private Label txStatusLabel;
+
+    private final String DEFAULT_NRG = "100000";
```

Comment:
`private final` field is named like `private static final` field - looks like a constant, but it's not.
Make these fields constants and move them above, with the other constants